compiler/AST/ImportStmt.cpp

```diff
@@ -66,6 +66,32 @@ ImportStmt::ImportStmt(BaseAST* source, const char* rename,
   gImportStmts.add(this);
 }
 
+ImportStmt::ImportStmt(BaseAST* source, bool isPrivate,
+                       std::vector<const char*>* namesList):
+  VisibilityStmt(E_ImportStmt) {
+
+  this->isPrivate = isPrivate;
+  this->modRename = astr("");
+  if (Symbol* b = toSymbol(source)) {
+    src = new SymExpr(b);
+
+  } else if (Expr* b = toExpr(source)) {
+    src = b;
+
+  } else {
+    INT_FATAL(this, "Bad mod in ImportStmt constructor");
+  }
+
+  if (namesList->size() > 0) {
```

Comment:
I probably wouldn't bother with this conditional (loop basically already has it)

Reply:
I'm thinking I'll leave this, it'll be useful when renaming the symbols in the {} list is online

Reply:
not a big deal